sdk/personalizer/Azure.AI.Personalizer/src/Generated/EvaluationsClient.cs

```diff
@@ -0,0 +1,227 @@
+        {
+            if (endpoint == null)
+            {
+                throw new ArgumentNullException(nameof(endpoint));
+            }
+            if (credential == null)
+            {
+                throw new ArgumentNullException(nameof(credential));
+            }
+
+            options ??= new PersonalizerBaseClientOptions();
+            _clientDiagnostics = new ClientDiagnostics(options);
+            string[] scopes = { "https://cognitiveservices.azure.com/.default" };
+            _pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, scopes));
+            RestClient = new EvaluationsRestClient(_clientDiagnostics, _pipeline, endpoint);
+        }
+
+        /// <summary> Initializes a new instance of EvaluationsClient. </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        public EvaluationsClient(string endpoint, AzureKeyCredential credential, PersonalizerBaseClientOptions options = null)
+        {
+            if (endpoint == null)
+            {
+                throw new ArgumentNullException(nameof(endpoint));
+            }
+            if (credential == null)
+            {
+                throw new ArgumentNullException(nameof(credential));
+            }
+
+            options ??= new PersonalizerBaseClientOptions();
+            _clientDiagnostics = new ClientDiagnostics(options);
+            _pipeline = HttpPipelineBuilder.Build(options, new AzureKeyCredentialPolicy(credential, "Ocp-Apim-Subscription-Key"));
+            RestClient = new EvaluationsRestClient(_clientDiagnostics, _pipeline, endpoint);
+        }
+
+        /// <summary> Initializes a new instance of EvaluationsClient. </summary>
+        /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
+        /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint. </param>
+        internal EvaluationsClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint)
+        {
+            RestClient = new EvaluationsRestClient(clientDiagnostics, pipeline, endpoint);
+            _clientDiagnostics = clientDiagnostics;
+            _pipeline = pipeline;
+        }
+
+        /// <summary> Delete the Offline Evaluation associated with the Id. </summary>
+        /// <param name="evaluationId"> Id of the Offline Evaluation to delete. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response> DeleteAsync(string evaluationId, CancellationToken cancellationToken = default)
```

Comment:
Based on usage, it may be helpful to provide a overload that takes an `IEnumerable<string> evaluationIds`.